windows/tools/OobSimulator/BindableBase.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
```

Comment:
nit: I think it would be nice to use \#ifdef guards instead of pragma, just like the reset of the nearobject-framework